### ISSUE TITLE
Code fork

### ISSUE THREAD

**Issue by avysk** [Conservatory/wmctrl#3]
For your information: I have forked the code at https://github.com/avysk/wmctrl-fork; added some enhancement from the net; and created a FreeBSD port.

**Comment by juhp** [comment]
Could you enable the issue tracker?

Why rename to `wmctrl-fork`?

**Comment by avysk** [comment]
Issue tracker enabled. The rename was requested by FreeBSD people who committed my port (I have no commit rights myself) into the tree (think about "port" as a Makefile, describing where to get the code, how to compile it, and how to install). There is so-called `MOVED` file in FreeBSD port tree, so anyone who used the original (not supported, non-buildable now) port will automatically change to mine.

The binary is still called `wmctrl`; the rename request was only for the repository name.

**Comment by juhp** [comment]
Thanks  (I am the fedora maintainer btw 👋)

Fair enough - I dunno I mean I kinda get it, but why not just switch directly to your upstream...
I think in Fedora our source packages are supposed to follow upstream name - but I don't have the time or energy to add a new package for this really, but maybe someone who needs it could do so - I need to check if it is okay.

**Comment by avysk** [comment]
I also was puzzled a bit by request (the repo was called originally just `wmctrl`) but I decided it is not a battle I want to fight :)

**Comment by juhp** [comment]
Anyway probably I could switch the fedora wmctrl package to follow wmctrl-fork.

**Comment by juhp** [comment]
Well I don't want to insist, but I really think it would make more sense to rename it back to wmctrl 🤷‍♂️ 
(it is a bit like calling one's `main` branch `main-branch`).
It is also clearly a github fork anyway...
I don't think it is really a downstream's prerogative to decide the upstream name either: though they can choose the name of their packaging of course.

(The alternative would be to do a hard rename like wmctrl2 or something completely different - though I am not advocating that.)

**Comment by avysk** [comment]
I agree with you, but as said, I felt it is not the battle I want to fight in. I wanted the port in FreeBSD, so I just complied.
